CONTRIBUTING.md

```diff
@@ -115,6 +115,9 @@ on GitHub consists of several projects:
  -  *Libplanet.Explorer.Executable*: Turns Libplanet Explorer into a single
     executable binary so that it is easy to distribute.
 
+ -  *Libplanet.Unity*: Libplanet unity plugin
+    This is a temporary plugin and will be transferred to sdk later.
```

Comment:
```suggestion
 -  *Libplanet.Unity*: User-friendly pluggable API for developing and running
    a *Libplanet* blockchain node within Unity.
```
```

Reply:
I don't think phrases such as "temporary" and "will be moved later" are appropriate. 😶